src/pages/index.astro

```diff
@@ -175,5 +253,90 @@ section.hero {
     flex: 1 1 auto;
     align-items: stretch;
   }
+
+  .updates-heading {
+    font: var(--font-h4);
+    text-align: center;
+  }
+
+  .updates-dots {
+    margin-bottom: var(--spacing-lg);
+  }
 }
 </style>
+
+<script>
+  const updatesContainer = document.querySelector('#updates-list');
+  const prevButton = document.querySelector('#prev-button');
+  const nextButton = document.querySelector('#next-button');
+  const carouselDots = document.querySelectorAll('#updates-dots span');
+  const carouselLinks = document.querySelectorAll<HTMLAnchorElement>('a.carousel-links');
+  const numberOfUpdates = carouselDots.length;
+
+  if (updatesContainer instanceof HTMLElement) {
+    let width = updatesContainer.offsetWidth;
+
+    window.addEventListener('resize', () => {
+      width = updatesContainer.offsetWidth;
+    });
+
+    prevButton?.addEventListener('click', () => scrollUpdates('left'));
+    nextButton?.addEventListener('click', () => scrollUpdates('right'));
+    
+    carouselDots[0].classList.add('active-dot');
+    updateTabIndices(0); 
+
+    function scrollUpdates(direction: 'left' | 'right') {
+      const amount = direction === 'left' ? -width : width
+
+      updatesContainer?.scrollBy({
+        left: amount,
+        behavior: 'smooth',
+      });
+
+      updatesContainer?.addEventListener('scrollend', ()=> {
```

Comment:
Unfortunately the scrollend event has [limited support across browsers](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollend_event). So we'll need to find another approach.